src/main/java/org/candlepin/model/Consumer.java

```diff
@@ -778,58 +786,14 @@ public void setSystemPurposeStatusHash(String systemPurposeStatusHash) {
     }
 
     /**
-     * Fetches the ID of the environment with which this consumer is associated. If the consumer is not
+     * Fetches the ID of the highest prioritized environment with which
+     * this consumer is associated. If the consumer is not
      * associated with an environment, this method returns null.
      *
      * @return the ID of the environment for this consumer
      */
     public String getEnvironmentId() {
```

Comment:
Let's add a //TODO here that explains that this method is temporary and should be removed later on, not to forget

Reply:
Done.